bosh_cli/lib/cli/client/director.rb

```diff
@@ -602,6 +632,14 @@ def upload_and_track(method, uri, filename, options = {})
           file.stop_progress_bar if file
         end
 
+        def upload_without_track(uri, filename, options = {})
+          file = FileWithProgressBar.open(filename, 'r')
+          status, _ = post(uri, options[:content_type] || {}, file, options[:headers] || {}, options)
```

Comment:
We notice that you allow for `options[:headers]`, but, based on the two cases where the method `upload_without_track` is called, it is called without options (i.e. `options[:headers]` will always be nil)